docs/rules/filename-case.md

```diff
@@ -105,3 +105,20 @@ Don't forget that you must escape special characters that you don't want to be i
 	}
 ]
 ```
+
+### includePath
+
+Type: `boolean`\
+Default: `false`
+
+Checks the path for case style, in addition to the filename.
+
+```js
+"unicorn/filename-case": [
+	"error",
+	{
+		"case": "kebabCase",
+		"includePath": true,
```

Comment:
I don't think we need an option to enable the check. Should always enabled.

Reply:
That would make this a breaking change. Is that OK?

I agree that it makes more sense as a default, but perhaps it would then need an opt-out? Would we call that `excludePath` or `fileOnly`, or perhaps something else?

Reply:
Can this be an option that we add later? We could roll this out as a feature release, and if there's demand, we can change the default and extend the rule again later?

I can see that some people may also want to go down the path of having different rules for different directories - like `PascalCase` for React components, but `kebab-case` for other directories. That becomes quite complex to build out.

Reply:
We always do major release. https://github.com/sindresorhus/eslint-plugin-unicorn/releases